docs/starter-get-started.md

```diff
@@ -0,0 +1,48 @@
+# Get Started
+
+## What is a threat detection kit?
+
+If you are new to Forta, a threat detection kit will help you get started on monitoring and protecting Web3 projects and assets right away with little to no technical and security experience.
```

Comment:
I think no security expertise is a bit much. I would rephrase to 'no custom bot development needed to get security relevant value from Forta.' - security expertise to monitor these alerts is very much required.

Reply:
makes sense, updated.